hotkey.go

```diff
@@ -119,6 +119,90 @@ func (hk *Hotkey) String() string {
 	return s
 }
 
+var NamedKeyMap map[string]Key
+var NamedModifierMap map[string]Modifier
+
+//run at run-time
+func init() {
+	NamedKeyMap = map[string]Key{
+		"0":      Key0,
+		"1":      Key1,
+		"2":      Key2,
+		"3":      Key3,
+		"4":      Key4,
+		"5":      Key5,
+		"6":      Key6,
+		"7":      Key7,
+		"8":      Key8,
+		"9":      Key9,
+		"Space":  KeySpace,
+		"a":      KeyA,
+		"b":      KeyB,
+		"c":      KeyC,
+		"d":      KeyD,
+		"e":      KeyE,
+		"f":      KeyF,
+		"g":      KeyG,
+		"h":      KeyH,
+		"i":      KeyI,
+		"j":      KeyJ,
+		"k":      KeyK,
+		"l":      KeyL,
+		"m":      KeyM,
+		"n":      KeyN,
+		"o":      KeyO,
+		"p":      KeyP,
+		"q":      KeyQ,
+		"r":      KeyR,
+		"s":      KeyS,
+		"t":      KeyT,
+		"u":      KeyU,
+		"v":      KeyV,
+		"w":      KeyW,
+		"x":      KeyX,
+		"y":      KeyY,
+		"z":      KeyZ,
+		"Return": KeyReturn,
+		"Escape": KeyEscape,
+		"Delete": KeyDelete,
+		"Tab":    KeyTab,
+		"Left":   KeyLeft,
+		"Right":  KeyRight,
+		"Up":     KeyUp,
+		"Down":   KeyDown,
+		"F1":     KeyF1,
+		"F2":     KeyF2,
+		"F3":     KeyF3,
+		"F4":     KeyF4,
+		"F5":     KeyF5,
+		"F6":     KeyF6,
+		"F7":     KeyF7,
+		"F8":     KeyF8,
+		"F9":     KeyF9,
+		"F10":    KeyF10,
+		"F11":    KeyF11,
+		"F12":    KeyF12,
+		"F13":    KeyF13,
+		"F14":    KeyF14,
+		"F15":    KeyF15,
+		"F16":    KeyF16,
+		"F17":    KeyF17,
+		"F18":    KeyF18,
+		"F19":    KeyF19,
+		"F20":    KeyF20,
+	}
+
+	NamedModifierMap = map[string]Modifier{
+		"Ctrl":  ModCtrl,
+		"Shift": ModShift,
+		"Mod1":  Mod1,
+		"Mod2":  Mod2,
+		"Mod3":  Mod3,
+		"Mod4":  Mod4,
+		"Mod5":  Mod5,
+	}
+}
+
```

Comment:
Please also remove this. As I argued, I think this is work that should be done on the user side. If a package adds this, there will be many requests to add more. Also, This map is mutable. Hence, unsafe to be used correctly.